config/jobs/kubeflow/OWNERS

```diff
@@ -1,4 +1,9 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
+- abhi-g
 - jlewi
+- kunmingg
+- lluunn
+- richardsliu
+- texasmichelle
```

Comment:
nit; remove Michelle; I don't think she has context and I don't think she will be responsible for the test infra.

Reply:
done